src/main/resources/Development-documentation/gerrit-sceanrios.md

```diff
@@ -0,0 +1,224 @@
+# Gerrit scenarios
+
+These scenarios describes possible use cases that needs consideration during the
+implementation and update of the plugin
+
+<!-- TOC -->
+
+- [Gerrit scenarios](#gerrit-scenarios)
+  - [Create patch set - CASE1](#create-patch-set---case1)
+  - [Create patch set with rebase - CASE2](#create-patch-set-with-rebase---case2)
+  - [Create patch set with merge - CASE3](#create-patch-set-with-merge---case3)
+  - [Create patch set no previous events - CASE4](#create-patch-set-no-previous-events---case4)
+
+<!-- /TOC -->
+
+## Create patch set - CASE1
+
+Preconditions:
+
+- Previous review submits to  the master with hash `C0`
```

Comment:
I guess rephrasing "Previous review submits to" like "Previous review is submitted to" reads better.

Reply:
Rephrased in ee8d791. I hope it is better